youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -214,6 +230,83 @@ def _ffmpeg_filename_argument(self, fn):
         # Also leave '-' intact in order not to break streaming to stdout.
         return 'file:' + fn if fn != '-' else fn
 
+    def _resolve_postprocessor_arg_var(self, arg, info):
+        # map(lambda s: s % info, args)
+        if "%(" not in arg:
+            return arg
+
+        template_dict = dict(info)
+        template_dict['epoch'] = int(time.time())
+        # autonumber_size = self.params.get('autonumber_size')
```

Comment:
I copied the code from YoutubeDL and commented the lines that I thought were not applicable anymore
Just for easier review, I cleanup afterwards